glide.lock

```diff
@@ -608,7 +613,5 @@ testImports:
   version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
   subpackages:
   - difflib
-- name: github.com/stretchr/testify
-  version: f6abca593680b2315d2075e0f5e2a9751e3f431a
-  subpackages:
-  - assert
+- name: github.com/stretchr/objx
```

Comment:
Is this dependency actually needed? Did you run `glide up -v` to remove sub-vendors?

Reply:
Will check, I was using testify's mock lib briefly working on this and  I think that's what pulled it. It sucked though so it's gone now.